src/sentry/tsdb/snuba.py

```diff
@@ -123,8 +107,74 @@ class SnubaTSDB(BaseTSDB):
             "times_seen",
             [["outcome", "=", outcomes.Outcome.FILTERED]],
         ),
+        TSDBModel.project_total_received_ip_address: SnubaModelQuerySettings(
+            snuba.Dataset.Outcomes,
+            "project_id",
+            "times_seen",
+            [["reason", "=", FilterStatKeys.IP_ADDRESS]],
```

Comment:
Did not include ["outcome", "=", "outcomes.Outcome.FILTERED"] because it looks like these models can be incremented even if the outcome is not FILTERED: https://github.com/getsentry/sentry/blob/master/src/sentry/utils/outcomes.py#L87-L97

Reply:
The models in `FilterStatKeys` are only ever incremented if the outcome is filtered because that's the only time the increment call would ever pass a `reason` that is inside of `FilterStatKeys`.

Reply:
That being said, we don't need to include the outcome in the query. We'll know that the outcome is filtered, and we won't have a situation where `ip-address` or any other filter key reason is produced from a different outcome.